client/src/pages/private/CohortTable.js

```diff
@@ -33,11 +37,24 @@ export default ({ cohorts, editCohortAction }) => {
   };
 
   // Helpers
-
   const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
 
   const renderCell = (columnId, row) => {
     switch (columnId) {
+      case 'cohort_name':
+        return (
+          <Link
+            component='button'
+            variant='body2'
+            onClick={() => {
```

Comment:
Could this be extracted to a fn `handleCohortNav`?

Reply:
Good point!